test/common/formatter/substitution_formatter_speed_test.cc

```diff
@@ -75,7 +75,12 @@ BENCHMARK(BM_AccessLogFormatterSetup);
 
 // NOLINTNEXTLINE(readability-identifier-naming)
 static void BM_AccessLogFormatter(benchmark::State& state) {
-  MockTimeSystem time_system;
+  testing::NiceMock<MockTimeSystem> time_system;
+  ON_CALL(time_system, systemTime())
+      .WillByDefault(testing::Return(SystemTime(std::chrono::seconds(1234567890))));
+  ON_CALL(time_system, monotonicTime())
+      .WillByDefault(testing::Return(MonotonicTime(std::chrono::seconds(1234567890))));
+
```

Comment:
Since this block is repeated a lot, probably worth a helper function so you can just do `MockTimeSystem time_system = makeMockTime()` (the function can still wrap it in a NiceMock but you won't have to say it's so every time, or set up the ON_CALLs)

Reply:
Good ideal, let's have a try.